curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4ad.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 66c8ba975ee7230e29f6c4ad
+title: Step 6
+challengeType: 1
+dashedName: step-6
+---
+
+# --description--
+
+To check which loan a user is qualified for based on the `annualIncome` and `creditScore` they entered, you have to use `if/else if` statement, AKA conditonals.
+
+Here's the basic syntax of an `if/else if` statement:
+
+```js
+if (condition1) {
+  // Do something if condition1 is true
+} else if (condition2) {
+  // Do something else if condition1 is false and condition2 is true
+} else {
+  // Do something else if both condition1 and condition2 are false
+}
+```
```

Comment:
we can remove this because remember there are other workshops and labs before this where they practice working with conditionals.

Reply:
also, there are lecture videos that discuss this too. so they won't need the example here